persistence/src/main/java/org/cbioportal/genome_nexus/persistence/internal/VariantAnnotationRepositoryImpl.java

```diff
@@ -1,74 +1,6 @@
-/*
- * Copyright (c) 2016 Memorial Sloan-Kettering Cancer Center.
- *
- * This library is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY, WITHOUT EVEN THE IMPLIED WARRANTY OF MERCHANTABILITY OR FITNESS
- * FOR A PARTICULAR PURPOSE. The software and documentation provided hereunder
- * is on an "as is" basis, and Memorial Sloan-Kettering Cancer Center has no
- * obligations to provide maintenance, support, updates, enhancements or
- * modifications. In no event shall Memorial Sloan-Kettering Cancer Center be
- * liable to any party for direct, indirect, special, incidental or
- * consequential damages, including lost profits, arising out of the use of this
- * software and its documentation, even if Memorial Sloan-Kettering Cancer
- * Center has been advised of the possibility of such damage.
- */
-
-/*
- * This file is part of cBioPortal Genome Nexus.
- *
- * cBioPortal is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
- * published by the Free Software Foundation, either version 3 of the
- * License.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
-*/
-
 package org.cbioportal.genome_nexus.persistence.internal;
 
-import com.mongodb.DBObject;
-import org.cbioportal.genome_nexus.util.Transformer;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.data.mongodb.core.MongoTemplate;
-
-import java.util.List;
-
-/**
- * @author Selcuk Onur Sumer
- */
-public class VariantAnnotationRepositoryImpl implements VariantAnnotationRepositoryCustom
+public class VariantAnnotationRepositoryImpl extends GenericMongoRepositoryImpl
```

Comment:
We still need a class named `VariantAnnotationRepositoryImpl` in order spring boot to instantiate the repository at runtime